components/scream/src/share/field/field_layout.cpp

```diff
@@ -112,25 +112,21 @@ LayoutType get_layout_type (const std::vector<FieldTag>& field_tags) {
       // Possible scenarios:
       //  1) <CMP|TL,LEV|ILEV>
       //  2) <TL,CMP>
-      //  3) <TL>
-      //  4) <CMP1,CMP2>
+      //  3) <CMP1,CMP2>
       if ( (tags[1]==LEV || tags[1]==ILEV) && (tags[0]==CMP || tags[0]==TL)) {
         result = LayoutType::Vector3D;
       } else if ((tags[0]==CMP1 && tags[1]==CMP2) ||
-                 (tags[1]==CMP && tags[0]==TL) ||
-                  tags[0]==TL) {
+                 (tags[1]==CMP && tags[0]==TL)) {
         result = LayoutType::Tensor2D;
       }
       break;
     case 3:
       // The only scenarios are:
       //  1) <CMP1, CMP2, LEV|ILEV>
-      //  2) <TL,  LEV|ILEV>
       //  3) <TL,  CMP, LEV|ILEV>
```

Comment:
Would you mind also changing `3)` to `2)`, to avoid ppl wondering if we're missing something?